src/main/resources/bidder-config/openx.yaml

```diff
@@ -17,3 +17,8 @@ adapters:
       cookie-family-name: openx
       type: redirect
       support-cors: false
+      secondary:
+        url: https://u.openx.net/w/1.0/cm?id=891039ac-a916-42bb-a651-4be9e3b201da&ph=a3aece0c-9e80-4316-8deb-faf804779bd1&gdpr={{gdpr}}&gdpr_consent={{gdpr_consent}}&r=
+        redirectUrl: /setuid?bidder=openx&gdpr={{gdpr}}&gdpr_consent={{gdpr_consent}}&us_privacy={{us_privacy}}&uid=${OPENX_ID}
```

Comment:
Pls, remove `$` sign according to https://github.com/prebid/prebid-server/blob/7818efd3470aefbc3f13eab908074e3b3e4b64a3/static/bidder-info/openx.yaml#L18

Reply:
done